requirements/base.txt

```diff
@@ -12,7 +12,7 @@ git+https://github.com/edx/edx-submissions.git@1.1.4#egg=edx-submissions==1.1.4
 boto>=2.32.1,<3.0.0
 celery==3.1.18
 defusedxml==0.4.1
-django>=1.8.17,<1.9
+django<1.9a0 # 'a' resolves known bug on gemnasium
```

Comment:
@robrap and as for the second question, No. The last PR against this repo had '1.9' listed as the upper bound in lieu of '1.9.0'.